strapi/strapi
# feat(review-workflow): reduce join table name length

### What does it do?

It adds the possibility to shorten the name of the join table in attributes with relation type.

### Why is it needed?

Reduce the impact of too long names in database tables.

### How to test it?

- Launch the EE of Strapi
- See in DB tables with name {content-type-name}_strapi_rw_stage_links instead of {content-type-name}_strapi_reviewWorkflows_stage_links

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
